docs/creating-service-bindings/service-binding-resource.md

```diff
@@ -4,8 +4,122 @@ sidebar_position: 6
+    lastTransitionTime: "2020-10-15T13:23:23Z"
+    status: "True"
+    type: CollectionReady
+  - lastHeartbeatTime: "2020-10-15T13:23:36Z"
+    lastTransitionTime: "2020-10-15T13:23:23Z"
+    status: "True"
+    type: InjectionReady
+  - lastHeartbeatTime: "2020-10-15T13:23:36Z"
+    lastTransitionTime: "2020-10-15T13:23:23Z"
+    status: "True"
+    type: Ready
+  secret: binding-request-72ddc0c540ab3a290e138726940591debf14c581
+```
+
+### Conditions
+
+Service Binding Status Conditions have three types:
+
+| Condition | Description |
+| -------- | ----------- |
+| CollectionReady | It represents collection of secrets from the service. On failure of collecting secrets from the service, the `conditions.status` becomes "False" with a reason.
+ |
+| InjectionReady | It represents injection of secret into the application. On failure to inject information into the service, the `conditions.status` becomes "False" with a reason. |
+| Ready | It represents overall ready status. On failure, the `conditions.status` becomes "False" with a reason. |
+
+Conditions can have the following combination of type, status and reason:
+
+| Type            | Status | Reason               | Type           | Status | Reason                   | Type           | Status | Reason                    |
+| --------------- | ------ | -------------------- | -------------- | ------ | ------------------------ |----------------|--------|---------------------------|
+| CollectionReady | False  | EmptyServiceSelector | InjectionReady | False  |                          | Ready          | False  | EmptyServiceSelector      |
+| CollectionReady | False  | ServiceNotFound      | InjectionReady | False  |                          | Ready          | False  | ServiceNotFound           |
+| CollectionReady | True   |                      | InjectionReady | False  | EmptyApplicationSelector | Ready          | True   | EmptyApplicationSelector  |
+| CollectionReady | True   |                      | InjectionReady | False  | ApplicationNotFound      | Ready          | False  | ApplicationNotFound       |
+| CollectionReady | True   |                      | InjectionReady | True   |                          | Ready          | True   |                           |
+
+## Advanced Options
+
+| Property | Type | Description |
+| -------- | ---- | ----------- |
+| .spec.bindAsFiles | bool | When set as `false` enables injecting gathered bindings as env variables into the application/workload |
+| .spec.detectBindingResource | bool | The operator binds all information from dependent resources (secrets, configMaps, services) owned by backingService CR |
+| .spec.mounthPath | string | It is used to specify a custom binding root path |
+| .spec.mappings | array (object) | It is used to set customized binding secrets using a combination of Go and jsonpath templating |
+| .spec.namingStrategy | string | It is used to define injected environment variable key or file name |
```

Comment:
it should be explained in a dedicated sections, explaining when and why to use it.